core/domain/exp_domain.py

```diff
@@ -6515,7 +6516,7 @@ class ExplorationMetadataDict(TypedDict):
     param_specs: Dict[str, param_domain.ParamSpecDict]
     param_changes: List[param_domain.ParamChangeDict]
     auto_tts_enabled: bool
-    correctness_feedback_enabled: bool
+    correctness_feedback_enabled: Optional[bool]
```

Comment:
Why is this line needed at all?

Reply:
This was just to get information about the exploration, and I believe we can remove this too.

Reply:
Done.